Websites/browserbench.org/Speedometer3.0/resources/editors/assets/longscript.js

```diff
@@ -12256,7 +12256,7 @@
 
   var hydrationParentFiber = null;
```

Comment:
I don't think we should modify Speedometer 3 content since the benchmark has been released as is. Modifying content even just fixing spelling mistake can have unexpected effects on perf.